stanford-earth/SE3_BLT
# Earth 000

# READY FOR REVIEW

# Summary
- Updates to recent config
- Disables quick edit
- Where did the paragraph permission module go?

# Needed By (Date)
- Today pls.

# Steps to Test

1. Check out this branch
2. Composer update
2. Do a blt sync:refresh
3. Validate no config changes
4. Pary

# Affected versions or modules
- August release

# Associated Issues and/or People
- @pookmish 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
